cmd/kubelet/app/server.go

```diff
@@ -186,6 +186,120 @@ func Run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies) error {
 	return nil
 }
 
+// setKubeDepsClients initializes clients if not standalone mode and any of the clients are not provided
+func setKubeDepsClients(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, nodeName types.NodeName) error {
```

Comment:
instead of passing in the whole kubelet server object, just pass in the parameters you need

Reply:
ok